lib/src/connector/score_connector.dart

```diff
@@ -81,9 +81,14 @@ class ScoreConnector {
       parameter = ConnectorParameter(_scoreAllScoreUrl);
       parameter.data = data;
       result = await Connector.getDataByGet(parameter);
+      final List<String> evalQuestionnaireCheckTexts = ['教學評量', 'Course Evaluation Questionnaire'];
+      if (evalQuestionnaireCheckTexts.any((text) => result.contains(text))) {
+        //TODO: add notification to notify users to complete the Course Evaluation Questionnaire
+        throw RangeError("[TAT] score_connector.dart: evalQuestionnaireCheckTexts was found in request result");
```

Comment:
`RangeError` should not be used here. Either replace it with `FormatException`, `StateError` or define a custom exception ourselves.

Reply:
ok

Reply:
[48c30d4](https://github.com/NEO-TAT/tat_flutter/pull/214/commits/48c30d406158eeedd50d0e5f4bf60cab8d9b4498)